### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by yemo06**
Adding comments, ReadMe, and explicit albums function

### CODE REVIEW

File: CertifiedEnergy.py [hunk 1]
```diff
@@ -43,22 +43,40 @@ def getAlbumInfo(artistAlbumUriList):
         singleAlbum = spotify.get_album(artistAlbumUriList[album])
         albumName= singleAlbum['name']
         albumTracklen= singleAlbum['total_tracks']
-        newList=singleAlbum['tracks']['items']
-        explicitCount =getGenAttrDictfromList('explicit',newList).count(True)
+        # newList=singleAlbum['tracks']['items']
+        explicitCount =getGenAttrDictfromList('explicit',singleAlbum['tracks']['items']).count(True)
```
Comment (dawnwages) [inline CertifiedEnergy.py:47]: Why does the count have `True` in it?

Comment (yemo06) [reply CertifiedEnergy.py:47]: In spofity's JSON response the explicit filed in  the song info is noted by either 'True' or 'False' Im counting the amount of times in a album that a track is explicit by counting Trues

File: CertifiedEnergy.py [hunk 2]
```diff
@@ -43,22 +43,40 @@ def getAlbumInfo(artistAlbumUriList):
...
         albumName= singleAlbum['name']
         albumTracklen= singleAlbum['total_tracks']
-        newList=singleAlbum['tracks']['items']
-        explicitCount =getGenAttrDictfromList('explicit',newList).count(True)
+        # newList=singleAlbum['tracks']['items']
+        explicitCount =getGenAttrDictfromList('explicit',singleAlbum['tracks']['items']).count(True)
         albumInfo.append([albumName,albumTracklen,explicitCount,artistAlbumUriList[album]])
     return albumInfo
 
 ### Here is where the algorithm for filtering Non-Explicit albums should go ###
```
Comment (dawnwages) [inline CertifiedEnergy.py:51]: comments should be left like 

"""
    This is your comment.
"""

Comment (dawnwages) [reply CertifiedEnergy.py:51]: `
"""
    this is a multi line comment
    another line of comment
"""
`

File: ReadMething.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+    <br />
+    <br />
+    <a href="https://github.com/othneildrew/Best-README-Template">View Demo</a>
+    ·
+    <a href="https://github.com/othneildrew/Best-README-Template/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/othneildrew/Best-README-Template/issues">Request Feature</a>
+  </p>
+</div>
+
```
Comment (dawnwages) [inline ReadMething.md:51]: In the future, lets move from html elements to proper markdown! :) Looks great!

**Review by dawnwages — APPROVED** [review]
Looks great! I think it's awesome

**Pull request merged**